template-helpers/hbs-helpers.js

```diff
@@ -44,6 +44,71 @@ function recruitmentBanner(args) {
   return `<div class="recruitment-banner"><a id="recruitment-banner" href="${AppConstants.RECRUITMENT_BANNER_LINK}"  target="_blank" rel="noopener noreferrer" data-ga-link="" data-event-category="Recruitment" data-event-label="${escapeHtmlAttributeChars(AppConstants.RECRUITMENT_BANNER_TEXT)}">${AppConstants.RECRUITMENT_BANNER_TEXT}</a></div>`;
 }
 
+function microsurveyBanner(args) {
+  // don't show micro survey if we're already showing a recruitment banner
+  if (AppConstants.RECRUITMENT_BANNER_LINK && AppConstants.RECRUITMENT_BANNER_TEXT) {
+    return;
+  }
+
+  // don't show micro survey if language isn't English
+  // TODO: localize micro survey questions
+  if (!englishInAcceptLanguages(args)) {
+    return;
+  }
+
+  // don't show micro survey if user is not signed in
+  if (!args.data.root.req.session.user){
+    return;
+  }
+
+  const bannerOpeningDiv = "<div id=\"micro-survey-banner\" class=\"micro-survey-banner hidden\">";
+  const nowSecond = new Date().getTime() / 1000;
+  let surveyElements;
+  switch (nowSecond) {
```

Comment:
How does `nowSecond` work? I tried the snippet above in DevTools and `new Date().getTime() / 1000` returned `1614107470.262`, which would fail every `switch..case` below (except `default`). Did we only want to look at the last digit of the time?

Reply:
Can we wrap it in a `Math.floor()` or something?